capstone/src/main/webapp/scripts/poll.js

```diff
@@ -0,0 +1,137 @@
+const BAR_WIDTH = "690";
+const BAR_HEIGHT = "55";
+
+function getPollOptions() {
+  fetch("/poll")
+    .then(response => response.json())
+    .then(options => {
+      options.forEach(option => {
+        createOptionElementLoad(option);
+      });
+    });
+}
+
+function createOptionElementLoad(option) {
```

Comment:
This is already 55 lines long, can you maybe break it into a couple of functions?